tests/tasks/test_create_user.py

```diff
@@ -0,0 +1,146 @@
+from argilla.server.database import SessionLocal
+from argilla.server.models import User, UserRole
+from argilla.tasks.create_user import create_user
+from click.testing import CliRunner
+from pydantic import ValidationError
+
+
+@pytest.fixture
+def db():
+    with SessionLocal() as db, db.begin():
+        yield db
+
+        db.query(User).delete()
+        db.commit()
+
+
+def test_create_user_passing_all_params(db):
+    username = "test-user"
+    first_name = "First"
+    role = "admin"
+    password = "12345678"
+
+    runner = CliRunner()
+    cli_params = f"--username {username} --password {password} --role {role} --first-name {first_name}"
+    runner.invoke(create_user, cli_params)
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+    assert user.role.value == role
+    assert user.first_name == first_name
+    assert accounts.authenticate_user(db, username=username, password=password) == user
+
+
+def test_create_user_with_input_role(db):
+    username = "test-user"
+    role = "admin"
+
+    runner = CliRunner()
+    cli_params = f"--username {username} --password 12345678 --first-name First"
+    runner.invoke(create_user, cli_params, input=f"{role}\n")
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+    assert user.role.value == role
+
+
+def test_create_user_with_input_password(db):
+    username = "test-user"
+    password = "12345678"
+
+    runner = CliRunner()
+    cli_params = f"--username {username} --role admin --first-name First"
+    runner.invoke(create_user, cli_params, input=f"{password}\n{password}\n")
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+    assert accounts.authenticate_user(db, username=username, password=password) == user
+
+
+def test_create_user_with_role_default(db):
+    username = "test-user"
+    password = "12345678"
+
+    runner = CliRunner()
+    cli_params = f"--username {username} --password {password} --first-name First"
+    runner.invoke(create_user, cli_params, input=f"\n")
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+    assert user.role == UserRole.annotator
+
+
+def test_create_user_with_input_username(db):
+    username = "test-user"
+    password = "12345678"
+
+    runner = CliRunner()
+    cli_params = f"--password {password} --first-name First"
+    runner.invoke(create_user, cli_params, input=f"{username}\n")
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+
+
+def test_create_user_with_last_name(db):
+    username = "test-user"
+    password = "12345678"
+    last_name = "Last"
+
+    runner = CliRunner()
+    cli_params = f"--username {username} --password {password} --role admin --first-name First --last-name {last_name}"
+    runner.invoke(create_user, cli_params)
+
+    user = accounts.get_user_by_username(db, username)
+    assert user
+    assert user.last_name == last_name
+
+
+def test_create_user_with_wrong_username(db):
```

Comment:
We are using `invalid` instead of `wrong` in the rest of the tests.

```suggestion
def test_create_user_with_invalid_username(db: Session):
```